src/policy/lockfreeimmortalspace.rs

```diff
@@ -34,11 +36,11 @@ pub struct LockFreeImmortalSpace<VM: VMBinding> {
     /// start of this space
     start: Address,
     /// Total bytes for the space
-    extent: usize,
+    total_bytes: usize,
```

Comment:
What was wrong with "extent"?  I think "extent" and "total bytes" are synonyms.

Reply:
In `let VMRequest::Extent{ extent, top } = vmrequest else { unreachable!() };`, the `extent` is the extent for the virtual memory range. We use it for `args.heap.reserve(extent, top);`.

The field `extent` in the space is not virtual memory range. It is the actual size we may use. We use it for mmapping side metadata and initializing SFT. I was concerned that it may cause confusion to call it `extent`.